cli/chaosli/cmd/create.go

```diff
@@ -0,0 +1,485 @@
+
+	prompt := &survey.Input{
+		Message: query,
+		Help:    helpText,
+	}
+
+	var opt survey.AskOpt
+	if required {
+		opt = survey.WithValidator(survey.Required)
+	}
+
+	err := survey.AskOne(prompt, &result, opt)
+
+	if err != nil {
+		fmt.Printf("getInput failed: %v", err)
+	}
+
+	return result
+}
+
+func selectInput(query string, inputs []string, helpText string) (string, error) {
+	var result string
+
+	prompt := &survey.Select{
+		Message: query,
+		Options: inputs,
+		Help:    helpText,
+	}
+
+	err := survey.AskOne(prompt, &result)
+
+	return result, err
+}
+
+func getSliceInput(query string, helpText string, required bool) []string {
+	var results string
+
+	prompt := &survey.Multiline{
+		Message: query,
+		Help:    helpText,
+	}
+
+	var opt survey.AskOpt
+	if required {
+		opt = survey.WithValidator(survey.Required)
+	}
+
+	err := survey.AskOne(prompt, &results, opt)
+
+	if err != nil {
+		fmt.Printf("getSliceInput failed: %v\n", err)
+	}
+
+	return strings.Split(results, "\n")
+}
+
+func getDNS() v1beta1.DNSDisruptionSpec {
+	if !confirmKind("DNS Disruption", "Overrides DNS resolution for specified hostnames with a MitM DNS attack. All other DNS requests will use the target's normal DNS resolver.") {
+		return nil
+	}
+
+	getHostRecordPair := func() v1beta1.HostRecordPair {
+		hrPair := v1beta1.HostRecordPair{}
+
+		hrPair.Hostname = getInput("Specify a hostname to target",
+			"When your target makes a DNS request for this hostname; the disruption will make sure the value you specify is returned, rather than the real record.",
+			true,
+		)
+		hrPair.Record.Type, _ = selectInput("the type of DNS record to inject",
+			[]string{"A", "CNAME"},
+			"We only support these two types of DNS requests for now. An A record request gets back an IP for a hostname, while a CNAME request maps an alias domain name to the canonical name.")
+		helpText := "We're specifying an A record, so the value should be an IP address. You can specify multiple IP addresses, if desired. Simply delimit them with commas, no whitespace! The disruption will round-robin between the options."
+
+		if hrPair.Record.Type == "CNAME" {
+			helpText = "We're specifying a CNAME record, so the value should be a hostname to redirect to."
+		}
+
+		hrPair.Record.Value = getInput("What value would you like to inject into this DNS record?", helpText, true)
+
+		return hrPair
+	}
+
+	fmt.Println("Let's specify a DNS record to inject!")
+
+	spec := v1beta1.DNSDisruptionSpec{}
+	spec = append(spec, getHostRecordPair())
+
+	for confirmOption("Would you like to override another DNS record?", "") {
+		spec = append(spec, getHostRecordPair())
+	}
+
+	return spec
+}
+
+func getDiskPressure() *v1beta1.DiskPressureSpec {
+	if !confirmKind("Disk Pressure", "Applies IO pressure to the target") {
+		return nil
+	}
+
+	spec := &v1beta1.DiskPressureSpec{}
+
+	spec.Path = getInput(
+		"Specify a path to apply IO pressure to",
+		"Specify a specific mount point to target a specific disk",
```

Comment:
I think some examples here would be helpful, I know personally the Disk Pressure options can sometimes be confusing. A quick (e.g. /mnt/data) should suffice for example.